packages/react-native-client/scripts/lint_check.sh

```diff
@@ -14,6 +14,6 @@ cd ..
 
 echo "Running swift-format:check for react-native ios files \n"
 cd ios
-swift-format format -i -r ./ --configuration swift-format-config.json
 swift-format lint -s -r ./ --configuration swift-format-config.json
+swift-format format -i -r ./ --configuration swift-format-config.json
```

Comment:
Fun fact: order in command makes a difference. If we first format and then lint -> formatting issues won't be catch on CI, as all files will be formatted, so linter won't spot issues in them. And script will return `0` return code (everything is ok)